### PR TITLE
Add missing API calls for Docker 1.9+

### PR DESCRIPTION
**Pull request by ct16k**
Hello,

This is my quick attempt at fixing #21 #23. I've done some quick tests on CentOS 7.2 with docker 1.10.2 and ceph 9.2.0 and it seems to work for me.

The Get call is just a copy-paste of Path, with a slightly different output format, as per docs, while the List call is a simple implementation, with no error checking, as apparently make and append can't fail? (I'm a Go noob, this is actually my first time using it.)

I've also included some misc changes that seemed small/harmless enough to not warrant a different PR:
- updated contrib/el7/Makefile to the current Ceph release and proper location for the logrotate config (I guess the project was renamed?)
- updated the SPEC file from rbd-docker-plugin-rpm as well, to include the default logrotate.d configuration file

### CODE REVIEW

**Comment by hekaldama** [comment]
Thanks for this! Will take a look probably next Monday.

**Comment by a-nldisr** [comment]
Update?

**Comment by hekaldama** [comment]
Totally lost track of this. Sorry :(. Will look in the morning.

**Comment by a-nldisr** [comment]
np, was just a curious ping because im willing to try out the plugin but got no rush. 

Just read some of the code myself and maybe @ct16k can tell: You added infernalis in the ceph make file, is there any reason for this?

**Comment by ct16k** [comment]
I changed the version from hammer to infernalis only because that is the version of Ceph I tested the plugin against and I wanted to match the version of the plugin's dependencies with that of the cluster's services.

Unrelated, the commit I just pushed was to fix a conflict between the base branch and my fork. Serves me right for bundling multiple changes into a single PR.

**Comment by ghost** [comment]
It resolve all problems. Why still not merge?

File: contrib/el7/SPECS/rbd-docker-plugin.spec [hunk 1]
```diff
@@ -79,6 +83,9 @@ fi
 /usr/bin/systemctl daemon-reload >/dev/null 2>&1 || true
 
 %changelog
+* Thu Sep 10 2015 sheepkiller
```
Comment (hekaldama) [inline contrib/el7/SPECS/rbd-docker-plugin.spec:86]: You are not @sheepkiller, maybe change to @ct16k ?

File: driver.go [hunk 2]
```diff
@@ -357,6 +357,64 @@ func (d cephRBDVolumeDriver) Mount(r dkvolume.Request) dkvolume.Response {
...
+			Name: v.name,
+			Mountpoint: k,
+		})
+	}
+
+	log.Printf("INFO: List request => %s", vols)
+	return dkvolume.Response{Volumes: vols}
+}
+
+// Path returns the path to host directory mountpoint for volume.
```
Comment (hekaldama) [inline driver.go:387]: Should remove this.

File: driver.go [hunk 3]
```diff
@@ -357,6 +357,64 @@ func (d cephRBDVolumeDriver) Mount(r dkvolume.Request) dkvolume.Response {
 
 // Path returns the path to host directory mountpoint for volume.
```
Comment (hekaldama) [inline driver.go:358]: Should remove.

**Pull request merged**